tutorials-and-examples/nvidia-nim/blueprints/README.md

```diff
@@ -8,3 +8,14 @@ Here you will find the NVIDIA NIM Blueprints that can be provisioned to run on G
    - DiffDock
 
    You can follow the detailed steps [here](./drugdiscovery/README.md).
+
+2. [Digital Human for Customer Service](https://build.nvidia.com/nvidia/digital-humans-for-customer-service)
```

Comment:
I'd suggest putting this one first, given that I think it will have more general applicability than the bionemo one.